Features/Language.feature

```diff
@@ -0,0 +1,31 @@
+﻿Feature: Language
+
+Verify Languages functionality
+
+Background:
+	Given User at Profile Page
+
+@regression
+Scenario: Verify Choose Language Level has 4 options
+	And User click Language Tab
+	When Add New button is pressed
+	Then Choose Language dropdown has four level options
+
+Scenario: Verify user is able to add languages
+	And User click Language Tab
```

Comment:
your data on what language and skill you are adding should be passed through the feature file 
please change this

Reply:
also 
Please use tables  and pass values to your scenarios for example look at the below scenario 
This way you will get exposure to use them  and will know how to pass test data to the tests


Example 

	Given I logged into the portal
	When I click on tab Education
	And I add my education 
	Then I am able to see my education'

Would  change to 


@Education
Scenario Outline: 1-Add education
	Given I logged into the portal
	When I click on tab Education
	And I add my education including '<Country>', '<University>', '<Title>', '<Degree>', '<Graduation Year>'
	Then I am able to see my education details including '<Country>', '<University>', '<Title>', '<Degree>', '<Graduation Year>'

	Examples:
	| Country     | University                      | Title  | Degree  | Graduation Year |
	| New Zealand | Whitireia Community Polytechnic | M.Tech | Diploma | 2021            |
	| Venezuela   | Hhutech University              | B.A    | Diploma | 2011            |

See  https://cucumber.io/docs/gherkin/reference/

Reply:
Updated Scenario to Scenario Outline to use tables.